<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDialogConverterPreview</class>
 <widget class="QDialog" name="CDialogConverterPreview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>937</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>视频预览</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{background-color: rgb(100, 100, 100);}

QGroupBox
{
color:rgb(255,255,255);
font-size:16px;
font-family:Microsoft Yahei;
border:1px solid rgb(120,120,120);
border-radius:15px;
}
QLabel
{
color:rgb(255,255,255);
	font-size:15px;
	font-family:Microsoft Yahei;	
}
</string>
  </property>
  <widget class="QFrame" name="frameSysBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>937</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame
{
	border-image:url(:/Skin/bg-above2.png);
}
QLabel
{
border-image:null;
color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <property name="horizontalSpacing">
       <number>2</number>
      </property>
      <item row="0" column="1">
       <widget class="QLabel" name="labelTitle">
        <property name="maximumSize">
         <size>
          <width>170</width>
          <height>170</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel#labelTitle
{
	color: rgb(255, 255, 255);
	font-size:18px;
	font-family:Microsoft Yahei;	
}</string>
        </property>
        <property name="text">
         <string>视频预览</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="labelFileName">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>500</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="logo">
        <property name="minimumSize">
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/Skin/logo.png)</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QFrame" name="frame_sysmenu">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>122</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame
{border-image:null;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QPushButton" name="BtnClose">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>0</y>
           <width>30</width>
           <height>30</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>关闭</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	
	border-image: url(:/Skin/close.png)0 0 repeat

}
QPushButton:hover
{	
	border-image: url(:/Skin/close.png)30 0 repeat

}
QPushButton:pressed
{	
	border-image: url(:/Skin/close.png)60 0 0 0 repeat
}
QPushButton:disabled
{	
	border-image: url(:/Skin/close.png)90 0 0 0 repeat
}
</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBoxInfo">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>40</y>
     <width>221</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>文件信息</string>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>视频信息:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>音频信息:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>文件大小:</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>280</y>
     <width>221</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>转码基本设置</string>
   </property>
   <widget class="QComboBox" name="comboBoxFormat">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>87</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>avi</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>mp4</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>目标格式:</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxExtern">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>390</y>
     <width>231</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>高级设置</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>221</width>
      <height>161</height>
     </rect>
    </property>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>87</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>默认</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>视频参数:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>视频编码:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_4">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>130</y>
       <width>87</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>默认</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>72</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>音频编码:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>音频参数:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>屏幕大小:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_3">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>87</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>默认</string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxCut">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>570</y>
     <width>681</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>截取片段</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>641</width>
      <height>42</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>开始时间:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>结束时间:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>68</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QTimeEdit" name="timeEditStart">
       <property name="displayFormat">
        <string>H:mm:ss</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QTimeEdit" name="timeEditStop">
       <property name="displayFormat">
        <string>H:mm:ss</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="BtnTimeStart">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	
	border-image: url(:/Skin/control-button-stop.png)0 0  repeat

}
QPushButton:hover
{	
	border-image: url(:/Skin/control-button-stop.png)30 0 repeat

}
QPushButton:pressed
{	
	border-image: url(:/Skin/control-button-stop.png)60 0 0 0 repeat
}
QPushButton:disabled
{	
	border-image: url(:/Skin/control-button-stop.png)90 0 0 0 repeat
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QPushButton" name="BtnTimeStop">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	
	border-image: url(:/Skin/control-button-stop.png)0 0  repeat

}
QPushButton:hover
{	
	border-image: url(:/Skin/control-button-stop.png)30 0 repeat

}
QPushButton:pressed
{	
	border-image: url(:/Skin/control-button-stop.png)60 0 0 0 repeat
}
QPushButton:disabled
{	
	border-image: url(:/Skin/control-button-stop.png)90 0 0 0 repeat
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>630</y>
     <width>195</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <widget class="QPushButton" name="BtnOK">
      <property name="styleSheet">
       <string notr="true">
QPushButton{
color:rgb(255,255,255);
        background-color:rgb(6, 83, 131);
        border-width: 2px;
        border-color: rgb(150,150,150);
        border-style: solid;
        border-radius: 3;   
font-size:17px;   
	font-family:Microsoft Yahei;
		}
QPushButton:hover{  border-color: rgb(160,160,160);background-color:rgb(10, 89, 150);;}
QPushButton:pressed{padding: 3px;}
QPushButton:disabled{ background-color:rgb(110, 110, 110);;}</string>
      </property>
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="BtnCancel">
      <property name="styleSheet">
       <string notr="true">
QPushButton{
	color:rgb(255,255,255);
        background-color:rgb(6, 83, 131);
        border-width: 2px;
        border-color: rgb(150,150,150);
        border-style: solid;
        border-radius: 3;   
	font-size:17px;   
	font-family:Microsoft Yahei;
		}
QPushButton:hover{  border-color: rgb(160,160,160);background-color:rgb(10, 89, 150);;}
QPushButton:pressed{padding: 3px;}
QPushButton:disabled{ background-color:rgb(110, 110, 110);;}</string>
      </property>
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>681</width>
     <height>521</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid rgb(120,120,120);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>frameSysBar</zorder>
  <zorder>groupBoxInfo</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>groupBoxExtern</zorder>
  <zorder>groupBoxCut</zorder>
  <zorder>frame_2</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="sisecplayer.qrc"/>
 </resources>
 <connections/>
</ui>
